travis-ci/test-data/TARGET_IMAGINARYBOARD/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+##
```

Comment:
Please make this header match the format used in `src/mbed_tools/project/_internal/templates/CMakeLists.tmpl` (single `#`, no extra lines)

Reply:
Thanks for providing a fix for this @Patater